### PR TITLE
fix(pipeline): set latest-version extension as default

### PR DESCRIPTION
**Pull request by chengjoey**
#### What this PR does / why we need it:
set latest-version extension as default

#### Specified Reviewers:

/assign @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |   Fix the bug that set latest-version extension as default           |
| 🇨🇳 中文    |   将最新版本的extension设为默认           |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: apistructs/extention.go [hunk 1]
```diff
@@ -228,7 +228,8 @@ type ExtensionVersionQueryRequest struct {
 	Name       string
 	YamlFormat bool `query:"yamlFormat"`
 	//默认false查询有效版本, true查询所有版本
-	All string `query:"all"`
+	All                string `query:"all"`
+	OrderByVersionDesc bool   `query:"orderByVersionDesc"`
```
Comment (sfwn) [inline apistructs/extention.go:232]: bad arg.
dicehub can judge which version is default by itself, no need this arg.

**Review by sfwn — CHANGES_REQUESTED** [review]

**Comment by sfwn** [comment]
which pr caused the issue? please link it.

File: modules/dicehub/dbclient/extention.go [hunk 2]
```diff
@@ -282,14 +282,17 @@ func (client *DBClient) DeleteExtensionVersion(name, version string) error {
...
-		Where("name = ?", name)
+		Where("name = ?", req.Name)
 	// 不显式指定all=true,只返回public的数据
-	if all != "true" {
+	if req.All != "true" {
 		query = query.Where("public = ?", true)
 	}
+	if req.OrderByVersionDesc {
+		query = query.Order("version desc")
+	}
```
Comment (sfwn) [inline modules/dicehub/dbclient/extention.go:295]: do you need add `version asc` if `orderByVersionDesc` is false?

**Review by sfwn — CHANGES_REQUESTED** [review]

File: modules/dicehub/dbclient/extention.go (hunk 2 above)
Comment (chengjoey) [reply modules/dicehub/dbclient/extention.go:295]: if `orderByVersionDesc is false`, version is order asc
so don't need add `version asc`

**Review by sfwn — APPROVED** [review]

**Pull request merged**